tcms/testruns/templates/testruns/get.html

```diff
@@ -71,11 +71,21 @@ <h2 class="card-pf-title" style="text-align: left">
                         data-on-text="{% trans 'Running' %}" data-off-text="{% trans 'Finished' %}">
                 </h2>
 
+                <h2 class="card-pf-title" style="text-align: left">
+                    <span class="fa fa fa-calendar-o"></span>{% trans 'Planned start' %}:
+                    {{ object.planned_start}}
```

Comment:
Use a `|default:'-'` template filter in order to print a dash `-` instead of `None` when these fields don't have a value.